src/interfaces/IScorerResolver.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+/// @notice This contract is used to resolve attestations from EAS.
+/// The implementer should create a custom logic to fetch badges and attestations from EAS that
+/// don't follow the same standard, creating backwards compatibility with the Scorer.
+interface IScorerResolver is IBadge {
+  struct Badge {
```

Comment:
Should this struct be inside IBadge?

Reply:
This was a working in progress, it is not the real thing (yet)